src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  justify-content: center;
```

Comment:
You have `justify-content: center;` and `align-items: center;` in your `.stopwatch` class, but it doesn't seem to be a flex container. If you don't use flexbox or grid, these properties won't have any effect.